examples/shared/design-system/tests/setup/test-utils.tsx

```
@@ -0,0 +1,27 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const customRender = (ui: ReactElement, options?: any) =>
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+  render(ui, {
+    wrapper: AppTestProviders,
+    ...options,
+  });
+
+// re-export everything
+// eslint-disable-next-line import-x/export
+export * from '@testing-library/react';
+export { default as userEvent } from '@testing-library/user-event';
```

ESLint

[error] Unable to resolve path to module '@testing-library/user-event'.

(import-x/no-unresolved)